Connector/xmr/config.py

```diff
@@ -0,0 +1,90 @@
+#!/usr/bin/python
+from json import JSONEncoder
+from utils import utils
+
+
+class Config:
+
+    def __init__(self, coin, networkName):
+
+        self._coin = coin
+        self._networkName = networkName
+        self._protocol = ""
+        self._host = ""
+        self._rpcPort = 0
+        self._rpcPath = ""
```

Comment:
This should be reviewed in the future. We might need more properties for XMR

Reply:
This properties are the ones used right now. More can be add if it is necessary